README.md

```diff
@@ -643,340 +531,322 @@ terragrunt = {
         "refresh"
       ]
     }
+  }
+}
+```
```

Comment:
Updated the docs to make this clearer: https://github.com/gruntwork-io/terragrunt/pull/167/commits/5af50b0d64c2cf113a35458edbac78085597401c